tests/fixtures/fixtures/fixture10.json

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
No (i tried) because it requires the Article records to be split in 2 groups, with another model instance inbetween. None of the current fixtures cover this